WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -803,7 +804,7 @@
 	509.1.1 = {
 		birth = yes
 	}
-	550.1.1 = {
+	575.1.1 = {
```

Comment:
What is the reasoning behind this change?

Reply:
To avoid not having a character in the history of c_tyrconnell and c_tyrone

Reply:
But doesn't it remain blank after 575 in the years 575-600?

Reply:
No, 360530 fills the gap

Reply:
Ok